movies.py

```diff
@@ -0,0 +1,80 @@
+"""Movie ticket process system"""
+
+MOVIE_PRICE = 5.0
+
+def enough_money(customer):
+    """Check if customer has adequate funds"""
+    balance = customer["cash"]
+    if balance < MOVIE_PRICE:
```

Comment:
sorry, I should have seen this sooner, but if your logic is this simple, you can do some pretty sweet syntactic sugar in python: `return False if customer["cash"] < MOVIE_PRICE else True`